disputer/index.js

```diff
@@ -29,7 +30,8 @@ async function run() {
 
   // Client and dispute bot
   const empClient = new ExpiringMultiPartyClient(ExpiringMultiParty.abi, web3, emp.address);
-  const disputer = new Disputer(empClient, accounts[0]);
+  let gasEstimator = new GasEstimator();
```

Comment:
nit: why not `const`? seems like it'd be more consistent with other variable instantiations

Reply:
ye, good point. have pushed a fix.